test/mixin/download/download_file_on_web_mixin_test.dart

```diff
@@ -0,0 +1,160 @@
+import 'dart:io';
```

Comment:
i think we can ignore the .mocks.dart file

Reply:
What is the reason to ignore it? Can you share a little?

Reply:
IMO: 
- If we push `.mocks` files have many changes → No need to review it

Reply:
But if .mocks files are not pushed, there's local errors until we run build_runner. So IMO it's cleaner to push them but not review them. WDYT ?